content/page/privacy-policy.en.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Privacy policy"
+date: 2022-02-08 12:39:00
+draft: false
+---
+
+## Privacy Policy (Android)
+
+The εxodus Android application does not collect any personal information.
+
+The application requires the following permissions:
+
+* *INTERNET*: This permission is required so that the application can query the reports on the main εxodus instance <https://reports.exodus-privacy.eu.org/>. It is not used for any other purpose.
+* *ACCESS_NETWORK_STATE*: This permission allows us to know whether you are connected to the Internet before making any request.
+
+The application transmits the list of installed applications on the device, mixed with a number of random applications (not present on the device) in order to anonymize the list. This list is not kept by the εxodus server.
+
+On the εxodus API we keep some information for 16 days, for abuse prevention: IP address, date, time, requested URL and user-agent.
```

Comment:
I propose: `On the εxodus API server, we [...]`

Reply:
Did we have a french version?
Did we need a french version?

Reply:
εxodus is the platform, Exodus is the app and Exodus Privacy is the organistion. So the first pharse should be:
`The Exodus Applicaiton` no?